docs/getting-started/github-apps/github-apps.rst

```diff
@@ -66,6 +66,29 @@ to associate with a workflow on Dockstore. A template for both workflows and ser
 along with explanations for each field. For every branch/release on GitHub that has one of these files, a corresponding entry
 will be made on Dockstore.
 
+Error Handling
+----------------------------------
+Since Dockstore relies on GitHub to tell us when changes have been made on GitHub, there are chances that the message gets lost or delayed.
+Typically, Dockstore reacts within seconds of a change being made on GitHub, however service disruptions can delay this to a few minutes.
+If a message were to get lost, unfortunately you will need to push to GitHub again. Currently, there is no way to tell on Dockstore whether
```

Comment:
I'd drop this paragraph.

If we don't get  the message from GitHub, which I suspect is very rare, we wouldn't know that we didn't get it. :) So nothing is going to change in the future.

Reply:
This is older text that I hard time placing -- hearing that it's actually not really needed is great and will help shorten this section to the essentials. I'll bulldoze it. 🚧